src/templates/static/affiliates/signup.jsx

```diff
@@ -1,31 +1,206 @@
+
+                <BoxRow top_row>
+                    <Box title={it.L('Generous commission')}      image='commission-icon'       text={it.L('Choose a commission plan that suits your business strategy.')} />
+                    <Box title={it.L('On-time payments')}         image='payment-icon'          text={it.L('Get paid by the 15th of every month. We’ve never missed a payment since we started our programme in March 2004.')} />
+                    <Box title={it.L('Dedicated support')}        image='contact-icon'          text={it.L('Contact our dedicated affiliate support team for help and suggestions.')} />
+                </BoxRow>
+                <BoxRow bottom_row>
+                    <Box title={it.L('Advanced marketing tools')} image='marketing-icon'        text={it.L('Promote our products easily, and keep track of all your earnings with our nifty marketing tools.')} />
+                    <Box title={it.L('High conversions')}         image='conversion-icon'       text={it.L('We’ve paid millions in commission to date through our industry-leading affiliate programme.')} />
+                    <Box title={it.L('Multiple opportunities')}   image='diversify-income-icon' text={it.L('Diversify your income stream through other partnership opportunities such as the <a href=\'[_1]\'>Payment Agent Programme</a>.', it.url_for('payment-agent'))} />
+                </BoxRow>
+
+                <div className='gr-padding-30'>
+                    <a className='button' href={it.affiliate_signup_url}>
+                        <span>{it.L('Apply Now')}</span>
+                    </a>
                 </div>
             </div>
         </div>
+
+        <div className='container gr-padding-20'>
+            <div className='center-text'>
+                <h2>{it.L('Types of affiliate commission plans')}</h2>
+                <p>{it.L('You can choose from two types of affiliate commission plans:')}</p>
+            </div>
+    
+            <TabContainer className='gr-padding-20 full-width' theme='light'>
+                <TabsSubtabs
+                    id='commission_tabs'
+                    className='gr-parent tab-selector-wrapper'
+                    items={[
+                        { id: 'revenue_share',   text: it.L('Revenue share') },
+                        { id: 'turnover',        text: it.L('Turnover') },
+                        { id: 'commission_tabs_selector', className: 'tab-selector' },
+                    ]}
+                />
+                <div className='tab-content'>
+                    <TabContentContainer>
+                        <TabContent id='revenue_share'>
+                            <p>{it.L('Profit from increasingly higher payouts with tiered and laddered commision rates that reward you based on the net revenue generated by your referred clients.')}</p>
+                            <div className='gr-padding-20 center-text'>
+                                <table>
```

Comment:
any reason why you didn't use the `<Table/>` here?

Reply:
didn't know we had one 😄, just followed the style.binary.com